hudi-spark-datasource/hudi-spark2/src/main/java/org/apache/hudi/internal/DefaultSource.java

```diff
@@ -61,11 +61,11 @@ public DataSourceReader createReader(DataSourceOptions options) {
       DataSourceOptions options) {
     String instantTime = options.get(DataSourceInternalWriterHelper.INSTANT_TIME_OPT_KEY).get();
     String path = options.get("path").get();
-    String tblName = options.get(HoodieWriteConfig.TABLE_NAME.key()).get();
+    String tblName = options.get(HoodieWriteConfig.TABLE_NAME_VALUE.key()).get();
```

Comment:
`TABLE_NAME_VALUE` does not seem very intuitive to me. And then doing `TABLE_NAME_VALUE.key()`.

I wonder if using `TABLE_NAME_CFG` or `TABLE_NAME_CONF` was actually better, so we don't have think about coming up with alternative names and instead can just append this suffix to differentiate ? Just a thought.

Reply:
I'd like to avoid suffixes like _CFG or _CONF, given everything else here is a config as well and the class is named xxxConfig. 

I get your point though. Not a lot of great choices here. _TBL_NAME_ or just _TABLE_?

Reply:
`TBL_NAME` ?

Reply:
Going with `TBL_NAME`